enterprise/backend/test/metabase_enterprise/data_editing/api_test.clj

```diff
@@ -703,3 +703,106 @@
                                                   :params {:status "approved"}}
                                    :query-params {:dashcard-id (:id dashcard)}})
                              (select-keys [:status :body])))))))))))))
+
+(deftest list-and-add-to-dashcard-test
+  (let [list-req #(mt/user-http-request-full-response
+                   (:user % :crowberto)
+                   :get
+                   "ee/data-editing/tmp-action")]
+    (mt/with-premium-features #{:table-data-editing}
+      (mt/test-drivers #{:h2 :postgres}
+        (data-editing.tu/toggle-data-editing-enabled! true)
+        (with-open [test-table (data-editing.tu/open-test-table! {:id 'auto-inc-type
+                                                                  :text      [:text]
+                                                                  :int       [:int]
+                                                                  :timestamp [:timestamp]
+                                                                  :date      [:date]}
+                                                                 {:primary-key [:id]})]
+          (let [{:keys [status body]}
+                (list-req {})
+                all-actions  (:actions body)
+                crud-actions (filter #(= @test-table (:table_id %)) all-actions)
```

Comment:
Pedant has entered the chat.

How about `table-actions` rather? Cos there could be:

1) Other kinds of CRUD actions we're skipping (e.g. basic actions)
2) Actions for other tables?

Reply:
6599dec1c09b